.github/workflows/teslight-update-packaging-tool-and-tup-file-build.yml

```diff
@@ -0,0 +1,78 @@
+name: TesLight Update Packaging Tool and TUP file Build
+on:
+  push:
+    branches: ["main", "releases/**", "feat/DevOps_Auto_build_for_Update_Packages"]
```

Comment:
I guess last one should be removed once this is ready?

Reply:
can be removed, yeah